CHANGELOG.md

```diff
@@ -1,8 +1,35 @@
 ## v1.10.0
+This release introduces a number of important features and improvements:
 
-#### Other changes
+- ARM64 support
+- A new subcommand `rkt stop` to gracefully stop running pods
+- native Go vendoring with Glide
+- rkt is now packaged for openSUSE Tumbleweed and Leap
+
+#### New features
+- Add ARM64 support ([#2758](https://github.com/coreos/rkt/pull/2758)).
+- Replace Godep with Glide, introduce native Go vendoring ([#2735](https://github.com/coreos/rkt/pull/2735)).
+- rkt: rkt stop ([#2438](https://github.com/coreos/rkt/pull/2438)).
 
+#### Bug fixes
+- stage1/fly: respect runtimeApp App's MountPoints ([#2852](https://github.com/coreos/rkt/pull/2852)). Fixes #2846.
```

Comment:
Github will render them as correct links when this is merged, but I can change them to proper URLS here if desired.